packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -16,7 +22,6 @@
   * Added `overflowColumns` prop for columns that are scrollable.
   * Added support for pinned columns.
   * Added ability to dive into cells with focusable elements.
-  * Added ability to treat the `terra-worklist-data-grid` component as a single tabstop.
```

Comment:
According to the [Release commit](https://github.com/cerner/terra-framework/blame/2a6c19a5a3d868c0a7820ef99d483ef4fe8369cf/packages/terra-worklist-data-grid/CHANGELOG.md), this entry was added after the release and should be moved to Unreleased.